src/modules/privateChat/screens/chat.screen.tsx

```diff
@@ -31,21 +31,18 @@ import AppText from "../../common/components/typography/text.component";
 const ChatScreen = () => {
   const navigation = useNavigation() as any;
   const route = useRoute() as any;
-  const { user }: { user: any } = route.params;
+  const { doctor }: { doctor: any } = route.params;
```

Comment:
@oluwadamilarey 

The type of doctor should not be any, it's type User from the generated types